app/models/service_template_transformation_plan_request.rb

```diff
@@ -3,6 +3,20 @@ class ServiceTemplateTransformationPlanRequest < ServiceTemplateProvisionRequest
 
   delegate :transformation_mapping, :vm_resources, :to => :source
 
+  def load_transitions
+    self.state ||= 'pending'
+    {
+      :pending   => {'pending' => 'pending'},
+      :queued    => {'pending' => 'queued'},
+      :active    => {'pending' => 'active',
+                     'active'  => 'active',
```

Comment:
@hsong-rh 
Is active => active needed?

Reply:
Yes, one type of automated request needs it.

Reply:
@hsong-rh do you know which one of the automate request. Also if the next_state == current_state meaning active == active can't we just skip that state change